lib/weka/core/instances.rb

```diff
@@ -241,6 +262,24 @@ def instance_from(instance_or_values, weight:)
           DenseInstance.new(data, weight: weight)
         end
       end
+
+      def map_attribute_type(type)
+        type = type.downcase.to_sym
+        return nil unless Attribute::TYPES.include? type
```

Comment:
You don't need to return nil explicitly. `return unless` works just fine. Please also use parenthesis with `include?` in order to keep the style of method calls consistent within the project.